src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSiteJenkinsTest.java

```diff
@@ -24,388 +24,122 @@
-        }
-        
-        // Cannot delete not editable
-        {
-            UpdateSite site1 = new UpdateSite(
-                "test1",
-                "http://example.com/test/update-center.json"
-            );
-            DescribedUpdateSiteForDeleteTest target = new DescribedUpdateSiteForDeleteTest(
-                "test2",
-                "http://example.com/test2/update-center.json"
-            );
-            
-            // Multiple update site.
-            Jenkins.getInstance().getUpdateCenter().getSites().clear();
-            Jenkins.getInstance().getUpdateCenter().getSites().add(site1);
-            Jenkins.getInstance().getUpdateCenter().getSites().add(target);
-            
-            int initialSize = Jenkins.getInstance().getUpdateCenter().getSites().size();
-            
-            WebClient wc = new WebClient();
-            wc.setPrintContentOnFailingStatusCode(false);
-            
-            HtmlPage deleteSitePage = wc.goTo(String.format("%s/%s/delete", UpdateSitesManager.URL, target.getPageUrl()));
-            assertEquals("UpdateSite must not be deleted yet.", initialSize, Jenkins.getInstance().getUpdateCenter().getSites().size());
-            
-            HtmlForm deleteSiteForm = deleteSitePage.getFormByName("deleteSiteForm");
-            assertNotNull("There must be deleteSiteForm", deleteSiteForm);
-            
-            target.setEditable(false);
-            
-            try
-            {
-                System.out.println(deleteSiteForm.getActionAttribute());
-                submit(deleteSiteForm);
-                fail("This request must be rejected");
-            }
-            catch(FailingHttpStatusCodeException e)
-            {
-                assertEquals("This request must be rejected with 400 Bad Request", 400, e.getStatusCode());
-            }
-            assertEquals("UpdateSite must not be deleted.", initialSize, Jenkins.getInstance().getUpdateCenter().getSites().size());
-            
-            try
-            {
-                wc.goTo(String.format("%s/%s/delete", UpdateSitesManager.URL, target.getPageUrl()));
-                fail("This request must be rejected");
-            }
-            catch(FailingHttpStatusCodeException e)
-            {
-                assertEquals("This request must be rejected with 400 Bad Request", 400, e.getStatusCode());
-            }
-            assertEquals("UpdateSite must not be deleted.", initialSize, Jenkins.getInstance().getUpdateCenter().getSites().size());
-        }
+        UpdateSitesManager manager = new UpdateSitesManager();
+
+        assertThat("should register described UC only", manager.getManagedUpdateSiteList(), hasSize(1));
+        assertThat("should save class of UC",
+                manager.getManagedUpdateSiteList().get(0), (Matcher) isA(DescribedUpdateSiteForConfigureTest.class));

```

Comment:
You can test the instance (not its class) using `JenkinsRule#assertEqualDataBoundBeans`.